client/src/components/GameCard/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+let style = {
+  height: '10em',
+  'overflow-y': 'scroll',
+  transform: 'translateY(0%)'
+}
+
+let style2 = {
+  margin: '2%'
+}
+
+function GameCard (props) {
+  return (
+    <div className='row'>
+      <div className='right col s12'>
+        <li className='collection-item hoverable' key={props.name}>
+          <div>
+            <a href={props.officialsite} target='_blank'>
+              <img
+                className='left valign-wrapper'
+                style={style2}
```

Comment:
this is from an old commit. it needs to be changed to `let style = {
  marginRight: '8px'
}` and the img tag needs to be updated to just `style={style}`

Reply:
should be fixed